mashup-example/using-build-tool/qlik-sense-desktop/package.json

```diff
@@ -16,11 +16,11 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@nebula.js/sn-table": "1.11.0",
+    "@nebula.js/sn-table": "1.12.1",
```

Comment:
this is a bit weird, should it do this @LiKang6688 ?

Reply:
Looks correct from my side, in the mashup, the @nebula.js/sn-table was bumped to the latest every time when we did the release.